firebase-authexchange/src/main/kotlin/com/google/firebase/authexchange/network/Routes.kt

```diff
@@ -0,0 +1,105 @@
+package com.google.firebase.authexchange.network
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Defines what an HTTP request may look like when interacting with the backend API for
+ * [FirebaseAuthExchange][com.google.firebase.authexchange.FirebaseAuthExchange].
+ */
+@Serializable internal sealed interface Request
```

Comment:
This is probably just my lack of Kotlin knowledge -- but what is the purpose of having an empty interface here?

Reply:
Just to provide a parent class, and the `sealed` type. Because it's a sealed type, I could theoretically do something like such:
```Kotlin
<T: Request> fun handleRequest(request: T) =
  when (request) {
    is ExchangeOidcTokenRequest -> // ...
    is ExchangeCustomTokenRequest -> // ...
    is ExchangeInstallationTokenRequest -> // ...
  } // the compiler understands that these are all the possible subclasses of Request
```
Although it's generally just good functional design to have a parent class for common types.

Reply:
Gotcha, thanks for the explanation!